src/Framework.UnitTests/Microsoft.Build.Framework.UnitTests.csproj

```diff
@@ -63,6 +71,9 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Xml" />
```

Comment:
Please transform this into a nuget dependency instead (project.json file next to this csproj). Mixing nuget and these leads to confusing dependency resolution.

Reply:
Ah, that may explain some warnings I was seeing. Will change, thanks!